app/renderer/components/config_menu.js

```diff
@@ -39,22 +46,42 @@ module.exports = {
     </div>
     `,
 
-    methods: {
-        clearTasks() {
-            TaskConfig.clearTasks();
-            AppStatus.activeSuite = 0;
-        },
-        resetConfig() {
-            this.config.bottomBar=true;
-            this.config.animatedSpinner=true;
-        },
-        onClose() {
-            this.config.bottomBar = AppStatus.config.bottomBar;
-            this.config.animatedSpinner = AppStatus.config.animatedSpinner;
-        },
-        onSave() {
-            AppStatus.config.bottomBar = this.config.bottomBar;
-            AppStatus.config.animatedSpinner = this.config.animatedSpinner;
-        }
+  methods: {
+    exportTasks(){
+      let content = TaskConfig.getData() ;
+      content = {
+        "suites":content,
+        "version":"0.3.4"
+      };
+      content = JSON.stringify(content) ;
+      dialog.showSaveDialog(
+        {
+          defaultPath: require('os').homedir() + "/gtask.json",
+          filters: [
+            { name: 'json', extensions: ['json'] }
```

Comment:
json.json? to be honest I don't understand this "filters" bit

Reply:
not actually... name is not exactly the file name. 
![ss_20171008_131533](https://user-images.githubusercontent.com/15269938/31316293-d1168bcc-ac2a-11e7-93ef-a2b791d1910a.png)

Look at the img... the name parameter change not the file name but the "extension" name that appears next to the save buttom. (I'm not sure what it could be used for) 

I've tried to skip it and by default take the same value as extensions so we could just remove it... other option would be set it to "gtasks-json" to inform of what this JSON is for but I'm not sure if it would be confused.

Reply:
Ok, I understand, anyway the current behaviour looks fine to me, so Its ok to leave it at it is